cms/wp/db/backup/create.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * Create a Wordpress database backup.
+ *
+ * @package wordpress
+ */
+
+namespace Deployer;
+
+task(
+	'wp:db:backup:create',
+	function () {
+		// Ensure that the backup directory exists.
+		run( 'mkdir -p {{backups}}' );
+
+		within(
+			get( 'repo' ),
+			function () {
+                $date = date( 'Y-m-d--H-i-s' );
+                $filename = '{{hostname}}--' . $date . '.sql';
+				run( "{{wp}} db export {{backups}}/" . $filename . " --add-drop-table", array( 'timeout' => 60 * 20 ) );
+			}
+		);
+	}
+)->desc( 'Create a database backup file and gzip it.' )->once();
```

Comment:
The code style in these files doesn't match our PHP coding standard.  I think that's because `phpcs.xml.dist` is not configured to check the `cms` directory.  Would you mind updating that file to ensure we're using the same formatting everywhere?

Reply:
I have added commit 8975ebf with the necessary changes to ensure the code style is checked.  You can use `./vendor/bin/phpcbf` to fix most of the violations automagically.

Reply:
This is a little contentious.  It actually does match wordpress's style best practices. However those are very different than drupal's.    This code came from a repo setup to follow those wordpress code style standards (actual tabs -wordpress instead of 2 spaces - drupal).  I'm fine either way but I'd imagine if we are super strict then we'll have to provide two phpcs.xml files accordingly - especially if we want a phpcs test to pass on these files.

Reply:
If this code lived in a WordPress plugin I'd be completely fine with using their standards, but since it's for a "vanilla" PHP project I think we should leverage our existing PHP coding standard which is already in use elsewhere in this project (which coincidentally shares the same base standard as the Deployer project itself).  I don't think it will pose any problems to users who want to follow some other standard in their projects, as they won't (shouldn't) be enforcing their standard on third-party, Composer-installed dependencies like this one.

At the end of the day, I just want to ensure we aren't juggling multiple standards within this one codebase.